.github/workflows/ctag_DY_workflow.yml

```diff
@@ -80,10 +85,18 @@ jobs:
       run: |
         python -m BTVNanoCommissioning.utils.compile_jec Rereco17_94X jec_compiled
         
-    - name: ctag muon DY workflows
+    - name: ctag muon DY workflows with custom correction
+      run: |
+        python runner.py --workflow ctag_DY_sf --json metadata/test_w_dj_mumu.json  --limit 1 --executor futures -j 4 --isCorr --isJERC
+    
+    - name: ctag muon DY workflows with correctionlib
       run: |
-        python runner.py --workflow ctag_DY_sf --json metadata/test_w_dj_mumu.json  --limit 1 --executor iterative --isCorr --isJERC
+        python runner.py --workflow ctag_DY_sf --json metadata/nano_UL17_test.json  --limit 1 --executor futures -j 4 --campaign 2017_UL
```

Comment:
Out of curiosity: is there a specific reason to use futures instead of iterative as executor?

Reply:
Same for the switch to UL samples and to UL campaign. This is in general fine by me, but maybe we should instead have (a version with max 1000 events of) the samples stored at btv eos

Reply:
Ah I did not realize that those are central nanoAOD :) (as well as the part where the DeepJet inputs files are skipped for central Nano)